package.json

```diff
@@ -25,14 +25,12 @@
   },
   "homepage": "https://github.com/philipheinser/ember-lightning",
   "dependencies": {
-    "co": "^4.6.0",
-    "co-redis": "^2.1.0",
-    "hiredis": "^0.4.1",
-    "koa": "^2.0.0-alpha.2",
-    "redis": "^2.6.0-1"
+    "co-redis": "^2.1.1",
+    "koa": "^2.3.0",
+    "redis": "^2.7.1"
   },
   "engines": {
-    "node": "4.2.x"
+    "node": ">=7.6.0"
```

Comment:
[required](https://github.com/koajs/koa#installation) by koa for async function support.